Configuration/NodeTypes.yaml

```diff
@@ -34,11 +34,15 @@
             ul: true
           link:
             a: true
-          alignment: {  }
+          alignment:
```

Comment:
That's certainly not a good idea to add such presentational features to nodetypes. I'll not merge this.
Your options:
1) reconsider why you need this
2) override these setting in your site package
3) fork or start your own nodetypes package from scratch